lib/image_optim/worker/gifsicle.rb

```diff
@@ -32,6 +36,26 @@ def self.init(image_optim, options = {})
         OptionHelpers.limit_with_range(v.to_i, 1..3)
       end
 
+      LOSSY_OPTION =
+      option(:lossy, 0, 'lossy compression level (requires giflossy). `\
```

Comment:
I think it is better to use 0 for lossless and 20 for lossy by default, like `QUALITY_OPTION` for pngquant worker does. So that just enabling lossy compression overall or only for this worker already uses sane default value.

Reply:
Yes I would like this as well but when tried, the tests break as the pack doesn't have giflossy so the `--lossy` option breaks gifsicle when `allow_lossy` is true.

I guess the solution is to replace gifsicle with giflossy in the image_optim_pack but I'm not sure if you want to completely switch to giflossy. The current situation between two project is  misfortune.